lib/negroku/tasks/eye.rake

```diff
@@ -15,30 +15,44 @@ end
 
 namespace :eye do
 
-  task :start, [:process] do
-
+  desc "Loads eye config and starts monitoring"
+  task :load do 
+    on release_roles fetch(:eye_roles) do
+        execute "mkdir -p #{shared_path}/config/processes"
+        execute "cd #{current_path} && bundle exec eye load #{shared_path}/config/eye.rb"
+    end
   end
 
-  task :restart, [:process] do
-
+  desc "Starts monitoring"
```

Comment:
This is limited to start and stop one process at a time right?? It would be nice to be able to start the complete application with all it's processes definitions...